src/sagemaker/modules/train/model_trainer.py

```diff
@@ -203,6 +203,8 @@ class ModelTrainer(BaseModel):
         local_container_root (Optional[str]):
             The local root directory to store artifacts from a training job launched in
             "LOCAL_CONTAINER" mode.
+        remove_inputs_and_container_artifacts (Optional[bool]):
```

Comment:
This option seems to be specifically for local mode? Can we not just do the cleanup for local mode by default?
We should be very careful in expanding the input param list for ModelTrainer (we shouldn't unless User experience warrants it).

Reply:
+1

Is there any advantage we provide by keeping the artifacts stored?

Reply:
the users may need to check the environment variables and hyperparameters in each container to debug